src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -439,7 +439,12 @@ private function _process_directives( string $html ) {
 					array_pop( $tag_stack );
 				}
 			} else {
-				if ( 0 !== count( $p->get_attribute_names_with_prefix( 'data-wp-each-child' ) ) ) {
+				$each_child_attrs = $p->get_attribute_names_with_prefix( 'data-wp-each-child' );
+				if ( ! is_countable( $each_child_attrs ) ) {
```

Comment:
I'd prefer a `null` check here, it's more consistent with other HTML API code:

```suggestion
				if ( null === $each_child_attrs ) {
```